### PR TITLE
Cypress/E2E: Fixed plugin/marketplace TM4J keys

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Added keys to match TM4J

#### Ticket Link
NA

### CODE REVIEW

File: e2e/cypress/integration/plugins/marketplace/marketplace_spec.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ describe('Plugin Marketplace', () => {
             cy.get('.more-modal__list').scrollIntoView().should('be.visible');
         });
 
-        it('autofocus on search plugin input box', () => {
+        it('MM-T1947 autofocus on search plugin input box', () => {
```
Comment (prapti) [inline e2e/cypress/integration/plugins/marketplace/marketplace_spec.js:275]:
```suggestion
        it('MM-T2001 autofocus on search plugin input box', () => {
```

File: e2e/cypress/integration/plugins/marketplace/marketplace_spec.js [hunk 2]
```diff
@@ -492,7 +492,7 @@ describe('Plugin Marketplace', () => {
             cy.get('#error_bar').should('not.exist');
         });
 
-        it('display installed plugins', () => {
+        it('MM-T1985 display installed plugins', () => {
```
Comment (prapti) [inline e2e/cypress/integration/plugins/marketplace/marketplace_spec.js:495]:
```suggestion
        it('MM-T1979 display installed plugins', () => {
```
Don't this the 'local plugin' reference below really makes sense for this once, hence requesting a change to T1979 instead.

**Review by prapti — APPROVED** [review]
Approving, although there may need more improvement in how the tests are structured either in Cypress or in TM4J. However, all these tests might get replaced soon with the arrival of a new marketplace in the near future.
Thank you @josephbaylon!

File: e2e/cypress/integration/plugins/marketplace/marketplace_spec.js (hunk 1 above)
Comment (furqanmlk) [reply e2e/cypress/integration/plugins/marketplace/marketplace_spec.js:275]: @prapti both MM-T1947 and MM-T2001 look same to me, are these test cases duplicate ?

**Review by furqanmlk — APPROVED** [review]

**Comment by furqanmlk** [comment]
Thanks @josephbaylon , Looks good to me

**Pull request merged**